app/assets/javascripts/application.js

```diff
@@ -42,3 +42,42 @@ function show_notification_blood(msg){
 }
 
 
+function prepare_to_initialize(q) {
+  $(document).ready(function() {
+   google.maps.event.addDomListener(window, 'load',function () { initialize(q);});
+});
+}
+
+
+ function initialize(x) {
+
+  var input = document.getElementById(x);
+  var options = {
+    componentRestrictions: {country: "eg"}
+  };
+
+  var autocomplete = new google.maps.places.Autocomplete(input,options);
+
+ if (x == "searchTextField") {
+       google.maps.event.addListener(autocomplete, 'place_changed', function () {
+        var place = autocomplete.getPlace();
+        document.getElementById('addName').value = document.getElementById('searchTextField').value;
+        document.getElementById('addLat').value = place.geometry.location.lat();
+        document.getElementById('addLng').value = place.geometry.location.lng();
+   });
+  }
+
+else {
+    google.maps.event.addListener(autocomplete, 'place_changed', function () {
+     var place = autocomplete.getPlace();
+     document.getElementById('hospital_addName').value = document.getElementById('hospital_searchTextField').value;
+     document.getElementById('hospital_addLat').value = place.geometry.location.lat();
+     document.getElementById('hospital_addLng').value = place.geometry.location.lng();
+   });
+  }
+}
+
+
+ $(function() {
+  $( ".dtpicker" ).datepicker({changeYear: true, yearRange : '-100:+0',dateFormat: 'yy-mm-dd'});
+  });
```

Comment:
in app/views/users/registration/new.html.haml
can I change the name of class:'c-input' of field of searchTextField or
this is related to another issue?
and also can I change the class name of (f.text_field :birth_date)
,(f.text_field :lastdonation) in the same file?? to remove duplication of
date picker.
بتاريخ ٠٦‏/٠٦‏/٢٠١٦ ٢:٢٥ م، كتب "Ahmad Ajmi" <notifications@github.com>:

> In app/assets/javascripts/application.js
> <https://github.com/Health-Care/BloodDonation/pull/18#discussion_r65880899>
> :
>
> > +  }
> > +
> > +else {
> > +    google.maps.event.addListener(autocomplete, 'place_changed', function () {
> > +     var place = autocomplete.getPlace();
> > +     document.getElementById('hospital_addName').value = document.getElementById('hospital_searchTextField').value;
> > +     document.getElementById('hospital_addLat').value = place.geometry.location.lat();
> > +     document.getElementById('hospital_addLng').value = place.geometry.location.lng();
> > +   });
> > +  }
> > +}
> > +
> > +
> > + $(function() {
> > +  $( ".dtpicker" ).datepicker({changeYear: true, yearRange : '-100:+0',dateFormat: 'yy-mm-dd'});
> > +  });
>
> indentation
>
> —
> You are receiving this because you modified the open/close state.
> Reply to this email directly, view it on GitHub
> <https://github.com/Health-Care/BloodDonation/pull/18/files/a08fb7752af699997af7662f60c324cd1dc673a4#r65880899>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AQ43pg3pspzgcDxE7VjokAUzdQztHd4Iks5qJBHLgaJpZM4IR2y0>
> .
>